src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -527,7 +529,8 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 			this.table.rerenderGrid();
 		}));
 		if (this.enableFilteringFeature) {
-			this.filterPlugin = new HeaderFilter(this.contextViewService);
+			this.filterPlugin = new HeaderFilter(this.contextViewService, this.notificationService);
+			this.filterPlugin.disabledMessage = localize('resultsGrid.maxRowCountExceeded', "Max row count for filtering/sorting has been exceeded. To update it, you can go to User Settings and change the setting: 'queryEditor.results.inMemoryDataProcessingThreshold'");
```

Comment:
Usually I would expect stuff like this to be passed in via an optional options object